package.json

```diff
@@ -111,6 +111,8 @@
     "flatpickr": "4.2.2",
     "lodash.debounce": "^4.0.8",
     "lodash.isequal": "^4.5.0",
+    "react-router": "^4.2.0",
```

Comment:
Are these dependencies or do we just need them in `devDependencies`?

Reply:
So I basically just added this to test that it would work with react-router, but what we could do is to remove it and then add a Code Sandbox example instead showing people how to use it with `react-router`

Reply:
If we move it to `devDependencies` seems like it'd work with what you have already, right? And then it's not included in the prod dependencies for the package.

Reply:
I'd prefer to just not have it added though - if anything we can add it to the add-on repo instead 🙂